src/features/bookings/pages/Bookings/Bookings.tsx

```diff
@@ -52,20 +52,24 @@ export const Bookings = () => {
     'réservations'
   )
 
-  const updateReactions = useCallback(() => {
-    const bookingsToUpdate =
-      endedBookings
-        .filter((endedBooking) => endedBooking.userReaction === null)
-        .map((booking) => booking.stock.offer.id) ?? []
-
-    const mutationPayload = bookingsToUpdate.map((bookingId) => ({
-      offerId: bookingId,
-      reactionType: ReactionTypeEnum.NO_REACTION,
-    }))
-    if (mutationPayload.length > 0) {
-      addReaction({ reactions: mutationPayload })
+  const updateReactions = useCallback(async () => {
+    if (isPending) return
+
+    const reactableEndedBookings = endedBookings.filter(
+      (endedBooking) => endedBooking.canReact === true && endedBooking.userReaction === null
+    )
+
+    const reactionRequest: PostReactionRequest = {
+      reactions: reactableEndedBookings.map((b) => ({
+        offerId: Number(b.stock.offer.id),
+        reactionType: ReactionTypeEnum.NO_REACTION,
+      })),
+    }
+
+    if (reactionRequest.reactions.length > 0) {
```

Comment:
if (reactionRequest.reactions.length > 0)
```suggestion
    if (reactionRequest.reactions.length) {
```